fast64_internal/oot/tools/quick_import.py

```diff
@@ -97,19 +108,22 @@ def quick_import_exec(context: bpy.types.Context, sym_name: str):
         else:
             settings.mode = "Generic"
             settings.name = sym_name
-            settings.folder = object_name
+            settings.folder = folder_name
             settings.actorOverlayName = ""
         bpy.ops.object.oot_import_skeleton()
     elif sym_def_type == "AnimationHeader" and not is_array:
         settings: OOTAnimImportSettingsProperty = context.scene.fast64.oot.animImportSettings
         settings.isCustom = False
         settings.isLink = False
         settings.animName = sym_name
-        settings.folderName = object_name
+        settings.folderName = folder_name
         bpy.ops.object.oot_import_anim()
+    elif sym_def_type == "CutsceneData" and is_array:
+        path = assets_scenes_dir_p / folder_name / sym_file_p
+        bpy.context.scene.ootCSNumber = importCutsceneData(f"{path}", None, sym_name)
```

Comment:
did you test this? `sym_file_p` should be the path to file itself, no need to prepend anything.
If you were lucky and `sym_file_p` is absolute I guess it is equivalent to `path = sym_file_p` and would still work

Reply:
yeah I tried it with that tiny cutscene in hyrule field that plays after the cutscene where you learn song of time and it worked